.github/workflows/test-e2e.yaml

```diff
@@ -1,7 +1,6 @@
 name: E2E Test
 
 on:
-  - push
```

Comment:
@tenzen-y I removed the e2e trigger on push since it is not needed.
Also, the large runners won't work on forks, so it is better to remove it: https://github.com/andreyvelich/trainer/actions/runs/13681493177

Reply:
At least, it would be better to perform those in the release branch.
But, I'm ok with addressing it in a follow-up.